index.rst

```diff
@@ -162,60 +162,68 @@ I assume that targetId would become target_id at the ConsDB (?).
 
 
 .. list-table:: Observatory Metadata
-   :widths: 25 25 50 25 25 25 25
+   :widths: 25 25 25 50 25 25 25 25
    :header-rows: 1
 
    * - FBS
      - JSON BLOCK
-     - CSC
+     - nextVisit
+     - Final CSC
      - FITS header
      - ObservationInfo
      - Butler exposure records
      - ConsDB
    * - science_program
      - program
+     - survey
      - Camera.startIntegration.additionalValues
      - PROGRAM
      - science_program
      - science_program
      - science_program
    * - observation_reason
      - reason
+     - --
```

Comment:
I think we can add "needed" for all of these rows based on Eric's comment at the meeting yesterday.

Reply:
Done (although didn't add target-id or scheduler-note as 'needed').